src/wmnds/components/content-tiles/advice-moving/advice-moving.njk

```diff
@@ -0,0 +1,51 @@
+{% macro wmndsContentTileAdviceMoving(params) %}
+<div class="wmnds-content-tile">
+   <div class="wmnds-content-tile__title">
+      <h2>Advice to keep moving</h2>
+   </div>
+   
+   <div class="wmnds-content-tile__buttons">
+      <button class="wmnds-btn wmnds-btn--primary wmnds-content-tile__buttons--button">
+         I drive
+         <svg class="wmnds-btn__icon wmnds-btn__icon--right">
+            <use xlink:href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right" href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right"></use>
+         </svg>
+      </button>
+
+      <button class="wmnds-btn wmnds-btn--primary wmnds-content-tile__buttons--button">
+         I use the bus
+         <svg class="wmnds-btn__icon wmnds-btn__icon--right">
+            <use xlink:href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right" href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right"></use>
+         </svg>
+      </button>
+
+      <button class="wmnds-btn wmnds-btn--primary wmnds-content-tile__buttons--button">
+         I use the train
+         <svg class="wmnds-btn__icon wmnds-btn__icon--right">
+            <use xlink:href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right" href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right"></use>
+         </svg>
+      </button>
+
+      <button class="wmnds-btn wmnds-btn--primary wmnds-content-tile__buttons--button">
+         I use the tram
+         <svg class="wmnds-btn__icon wmnds-btn__icon--right">
+            <use xlink:href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right" href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right"></use>
+         </svg>
+      </button>
+
+      <button class="wmnds-btn wmnds-btn--primary wmnds-content-tile__buttons--button">
+         I walk or cycle
+         <svg class="wmnds-btn__icon wmnds-btn__icon--right">
+            <use xlink:href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right" href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right"></use>
+         </svg>
+      </button>
+
+      <button class="wmnds-btn wmnds-btn--primary wmnds-content-tile__buttons--button">
+         I want to visit an attraction or venue
+         <svg class="wmnds-btn__icon wmnds-btn__icon--right">
+            <use xlink:href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right" href="/img/wmnds-sprite.min.svg#wmnds-general-chevron-right"></use>
+         </svg>
+      </button>
```

Comment:
Call a button in using the nunjucks import syntax.
This keeps the code clean, and this ensures that when the primary button is updated, so is this component.
I've put an example import below....

```suggestion
      {%- from "wmnds/components/button/_button.njk" import wmnButton as primary -%}
      {{
        primary({
          classNames: "wmnds-btn--primary wmnds-content-tile__button",
          text: "I walk or cycle",
          iconRight: "general-chevron-right"
        })
      }}
```